probes/commitsInLast90Days/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: commitsInLast90Days
+short: Check whether the project has at least one commit per week over the last 90 days.
+motivation: >
+  A project which is not active might not be patched, have its dependencies patched, or be actively tested and used. However, a lack of active maintenance is not necessarily always a problem. Some software, especially smaller utility functions, does not normally need to be maintained. For example, a library that determines if an integer is even would not normally need maintenance unless an underlying implementation language definition changed. A lack of active maintenance should signal that potential users should investigate further to judge the situation. A project may not need further features or maintenance; In this case, the probe results can be disregarded.
+implementation: >
+  The implementation checks the number of commits made in the last 90 days by any user type.
+outcome:
+  - If the project has commits from the last 90 days, the probe returns as many OutcomePositive as the project has commits.
+  - If the project does not have commits in the last 90 days, the probe returns a single OutcomeNegative.
```

Comment:
will need tweaked if we start using `Values`

Reply:
Done in https://github.com/ossf/scorecard/pull/3507/commits/e53f4dfdff0cec59cb9d0486bf70e27b3ae3e0ca